### PR TITLE
Create Experience with GCI 2017

### PR DESCRIPTION
**Pull request by abhinavraj0123456789**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [ ] Resolved merge conflicts
- [ ] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please see the readme on how to add your blog.

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@abhinavraj0123456789  Please read README.md for adding the blog 👍

File: Experience with GCI 2017 [hunk 1]
```diff
@@ -1 +1,4 @@
-Preview my blog- https://gcitour2017.wordpress.com/2017/12/21/gci-tour-2017/ 
+- url: https://gcitour2017.wordpress.com/2017/12/21/gci-tour-2017/
+  title: GCI TOUR 2017
+  img: <img class="alignnone size-full wp-image-19" src="https://gcitour2017.files.wordpress.com/2017/12/img_20171203_210923294.jpg" alt="IMG_20171203_210923294.jpg" width="3024" height="4032" />
+  author_name: Abhinav Raj
```
Comment (abishekvashok) [inline Experience with GCI 2017]: It should be added to blogs.yml 
img: should contain the images name

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: Experience with GCI 2017 [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
```
Comment (abishekvashok) [inline Experience with GCI 2017]: Please add to blogs.yml img name should be your images name.

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Review by nikit19 — CHANGES_REQUESTED** [review]
You have to add these details in  'blogs.yml' in _data folder.

**Review by 0x48piraj — COMMENTED** [review]
Set a suitable PR title!

Hint: "What you did to the codebase"

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@abhinavraj0123456789  I think you are adding blog, so please follow `README.md` on how to add blog and update your PR 👍

**Comment by supunTE** [comment]
If you are going to add a blog add it to blogs.yml as readme.md

**Comment by harshit98** [comment]
Hi @abhinavraj0123456789 , what's the status of the pull request?

**Review by ParthS007 — COMMENTED** [review]
closing this PR due to inactivity, Please create a new PR after go through `README.md` , Thanks

**Pull request closed without merging**